src/main/resources/schemas/nanopore-instrument-output.schema.json

```diff
@@ -390,6 +390,9 @@
                   {
                     "$ref": "#/definitions/duty_time_log"
                   },
+                  {
+                    "$ref": "#/definitions/barcode_alignment_log"
+                  },
```

Comment:
Are you sure this will be needed in the old schema as well? Mibi never communicated this requirement to us

Reply:
I agree, I will remove it